superset-frontend/src/dashboard/components/PropertiesModal.jsx

```diff
@@ -338,7 +339,13 @@ class PropertiesModal extends React.PureComponent {
               >
                 {saveLabel}
               </Button>
-              <Button type="button" buttonSize="sm" onClick={onHide} cta>
+              <Button
+                data-test="cancel-button"
```

Comment:
This makes me a little nervous since there must be a hundred cancel buttons in the app. Something like `cancel-properties-modal-button` would be much more explicit and easy to trace.